File: java-service/src/main/java/com/slimblog/entity/AdminUser.java
package com.slimblog.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import lombok.Getter;
import lombok.Setter;

/**
 * <p>
 * 
 * </p>
 *
 * @author yzq
 * @since 2022-03-19
 */
@Getter
@Setter
@TableName("admin_user")
public class AdminUser implements Serializable {

    private static final long serialVersionUID = 1L;

    @TableId(value = "Id", type = IdType.AUTO)
    private Integer id;

    private String username;

    private String password;

    private String nickname;

    private String email;

    private String phone;

    private String role;


}

File: kotlin-service/src/main/kotlin/com/slimblog/model/entity/AdminUser.kt
package com.slimblog.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId

/**
 * @author yanni
 * @date time 2022/3/19 21:34
 * @modified By:
 *
 */
data class AdminUser(
    @TableId("id",type=IdType.AUTO)
    var id: String,
    var username: String,
    var password: String,
    var nickname: String,
    var email: String,
    var phone: String,
    var role: String
) {
}